### PR TITLE
Fix/toolbar hooks

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-2141

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/media-viewer/src/lib/media-viewer.component.ts [hunk 1]
```diff
@@ -56,6 +57,7 @@ export class MediaViewerComponent implements OnChanges, AfterContentInit {
 
   ngAfterContentInit() {
     this.setToolbarButtons();
+    this.toolbarEventsOutput.emit(this.toolbarEvents);
```
Comment (linusnorton) [inline projects/media-viewer/src/lib/media-viewer.component.ts]: Having something emitted just once on init feels a bit wrong but I guess that's the way we decided to implement this.

**Comment by linusnorton** [comment]
@velimd how come we've stopped using the subjects inside of the components and are copying the value to a local property instead?

File: projects/media-viewer/src/lib/annotations/annotation-set/annotation-set.component.ts [hunk 2]
```diff
@@ -45,6 +46,7 @@ export class AnnotationSetComponent implements OnInit, OnDestroy {
     this.subscriptions.push(this.viewerEvents.highlightedShape.subscribe((highlight) => this.initShapeRectangle(highlight.event)));
     this.subscriptions.push(this.annotationService.getSelectedAnnotation()
       .subscribe((selectedAnnotation) => this.selectedAnnotation = selectedAnnotation));
+    this.subscriptions.push(this.toolbarEvents.drawModeSubject.subscribe(drawMode => this.drawMode = drawMode));
```
[the remark this answers is not in the document]
Comment (velimd) [reply projects/media-viewer/src/lib/annotations/annotation-set/annotation-set.component.ts]: i thought it was best to change the service to a private variable, however this meant it wasnt being used in the html hence i added the variable. not sure but i thought this was the best way to do it than before.
Also before, it seemed like in the html, it wasnt checking the value but if the service existed or not.

File: projects/media-viewer/src/lib/media-viewer.component.ts (hunk 1 above)
Comment (velimd) [reply projects/media-viewer/src/lib/media-viewer.component.ts]: yh, i wasnt sure the best approach for this, i remember talking to you about it. i believe this is just so they can get the service so once is effort in my opinion.

File: projects/media-viewer/src/lib/annotations/annotation-set/annotation-set.component.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (velimd) [reply projects/media-viewer/src/lib/annotations/annotation-set/annotation-set.component.ts]: > @velimd how come we've stopped using the subjects inside of the components and are copying the value to a local property instead?

is this comment referring to this?

[the remark this answers is not in the document]
Comment (linusnorton) [reply projects/media-viewer/src/lib/annotations/annotation-set/annotation-set.component.ts]: yep

**Comment by velimd** [comment]
> Remove print test or fix?

not sure what you mean, i didnt include @ci for print as we discussed.

**Comment by jai-santhosh** [comment]
ci tests looks ok tome

**Pull request merged**